src/api/db/models/Project.js

```diff
@@ -8,12 +8,27 @@ import { sortDeps, hasRole, idMatch } from './utils.js';
 import { MLModelModel } from './MLModel.js';
 import retry from 'async-retry';
 import {
+  WRITE_PROJECT_ROLES,
   WRITE_DEPLOYMENTS_ROLES,
   WRITE_VIEWS_ROLES,
   WRITE_AUTOMATION_RULES_ROLES
 } from '../../auth/roles.js';
 
 export class ProjectModel {
+  static async queryById(_id) {
```

Comment:
Similar to the `MLModelModel`, do we need this? Shouldn't `getProjects({ _ids: [some_id]})` work?